packages/graphics/mesa/package.mk

```diff
@@ -13,8 +13,7 @@ PKG_LONGDESC="Mesa is a 3-D graphics library with an API."
 
 get_graphicdrivers
 
-PKG_MESON_OPTS_TARGET="-Ddri-drivers= \
```

Comment:
`-Ddri-drivers= \`
intentionally removed ?

Reply:
Yes - it was dropped in this release:
- https://gitlab.freedesktop.org/mesa/mesa/-/commit/59625a68ffc592cc5c6c62baf3a03853f234dbf2